pages/common/todoist.md

```diff
@@ -0,0 +1,28 @@
+# todoist
+
+> This program will let you use Todoist from the CLI.
+> More information: <https://github.com/sachaos/todoist>.
+
+- Add a task:
+
+`todoist add "{{some_task}}"`
+
+- Add a high priority task, with label, project, and due date:
+
+`todoist add "{{some_task}}" --priority {{1}} --label-ids "{{label_id}}" --project-name "{{project_name}}" --date "{{tmr 9am}}"`
+
+- Add the same task, in quick mode:
```

Comment:
Duplicating?

Reply:
the `quick` command calls the todoist api directly, this method uses the todoist-cli commands and options